src/hooks/useAliasPage.tsx

```diff
@@ -0,0 +1,34 @@
+import { PaymentsContext } from 'context/Payments';
+import { useRef, useContext, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { changeAliasLength } from '../utils/InputChange';
+const useAliasPage = () => {
+  const navigate = useNavigate();
+  const inputRef = useRef<HTMLInputElement>(null);
+  const [inputLength, setInputLength] = useState(0);
+  const paymentsCtx = useContext(PaymentsContext);
+  const cardList = paymentsCtx.cardList;
+  const currentCard = cardList.find((card) => card.id === localStorage.getItem('id'));
+
+  const handleSubmit = () => {
+    if (inputRef.current === null || !currentCard) return;
+    const aliasValue = inputRef.current.value;
+    const aliasName = !aliasValue.length ? currentCard.company : aliasValue;
```

Comment:
```suggestion
    const aliasName = aliasValue.length ? aliasValue : currentCard.company;
```
부정 조건을 상당히 많이 사용하시는데 사고의 흐름에는 헷갈림을 유발합니다
`Early Return`이 아닌 경우 긍정으로 비교해보는 것도 고려해보시는게 어떨까요?